testing/features/input.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Input
+  As a non-technical person at Citizens Advice
+  I want to know the full specification of what an “Input” component can / should do
+  So I can check whether it is being used correctly
+  And I can check if is buggy
+
+  Rule: Basic
+    Background:
+      Given I have a Basic Input component on page
+
+    Scenario: Input is on the page and I can type on it
+      Then there is a label
+      But there is no hint
+      And there is no optional field
+      And there is no error message
+      Then I can type "Hello" into the text box
```

Comment:
lets re-order these slightly so it reads better. If we place this one first, then the other four in the same order, I think it will read better

Reply:
I reorganized all the steps, so **Then** came first and **But** came last.